### ISSUE TITLE
Materialize crashed on Ubuntu 18.10

### ISSUE THREAD

**Issue by virgoss** [maikramer/Materialize#7]
Hello!
I tried Materialize-v0.4HDRP.beta8.Linux64.x86_64,  but it closes immediately as soon as I see the splash screen.

Materialize V03.5.1 run flawlessy.

This is the output from the terminal:

*firbint@firbint-desktop:~$ '/home/firbint/Scaricati/Materialize-v0.4HDRP.beta8.Linux64/Materialize-v0.4HDRP.beta8.Linux64.x86_64' 
Found path: /home/firbint/Scaricati/Materialize-v0.4HDRP.beta8.Linux64/Materialize-v0.4HDRP.beta8.Linux64.x86_64
Mono path[0] = '/home/firbint/Scaricati/Materialize-v0.4HDRP.beta8.Linux64/Materialize-v0.4HDRP.beta8.Linux64_Data/Managed'
Mono config path = '/home/firbint/Scaricati/Materialize-v0.4HDRP.beta8.Linux64/Materialize-v0.4HDRP.beta8.Linux64_Data/MonoBleedingEdge/etc'
Preloaded 'libStandaloneFileBrowser.so'
Preloaded 'ScreenSelector.so'
Display 0 'W2243 22"': 1920x1080 (primary device).
Loading player data from /home/firbint/Scaricati/Materialize-v0.4HDRP.beta8.Linux64/Materialize-v0.4HDRP.beta8.Linux64_Data/data.unity3d
Logging to /home/firbint/.config/unity3d/Editor/Player.log
Annullato (core dump creato)
firbint@firbint-desktop:~$ *

Thank you
Virginio

**Comment by maikramer** [comment]
Hi, thanks for your feedback, can you send me the Player.log?
Send me the following informations too.
GPU and driver being used.

HDRP version is based on vulkan and is too bleeding edge yet, since, Unity
on linux is preview, High Definition Renderer pipeline is preview, Unity
with Vulkan on linux is experimental, linux vulkan drivers still in a beta
stage, then, well, will take a bit to this version work as expected, but, i
can tell you, looks so nice.

Read more here:
https://github.com/maikramer/Materialize/blob/HDRP/README.md

[image: Petróleo Brasileiro S.A.] <http://www.petrobras.com.br/pt/>

Maikeu Locatelli / Téc. Operação Pleno
maikramer@gmail.com / +55 27 99988-4345

Petróleo Brasileiro S.A.
+55 27 3048-9224
http://www.petrobras.com.br/pt/


Em dom, 12 de mai de 2019 às 13:41, virgoss <notifications@github.com>
escreveu:

**Comment by virgoss** [comment]
Hi Maikeu, this is the file, i hope help you.

[Player.log](https://github.com/maikramer/Materialize/files/3174634/Player.log)

GPU is Nvidia GeForce GTX970 4GB, driver version 390.116

**Comment by virgoss** [comment]
[Player.log](https://github.com/maikramer/Materialize/files/3174702/Player.log) (From Materialize-v0.4HDRP.beta8.Linux64.x86_64)

**Comment by maikramer** [comment]
Thanks, by the log the program got out of memory.
I will launch a new version in a hour or so, please, test and see if it's fixed.

**Comment by maikramer** [comment]
Try the new beta please, see if your problem has been fixed. Thanks a lot for your help.

**Comment by virgoss** [comment]
Hi Maikeu, Tested beta9, it work's, but sometimes freezing when switch to/from fullscreen or window mode

**Comment by maikramer** [comment]
Yes, that is a known bug, i'm looking with unity to fix it, i will Update
HDRP tomorrow and see it the problem is fixed. I'm working to reduce
greatly the memory usage also. Thanks for your help.

Em dom, 19 de mai de 2019 às 09:23, virgoss <notifications@github.com>
escreveu:

**Comment by virgoss** [comment]
Thank you!

**Comment by maikramer** [comment]
Can you try the 0.3.7 version and see if it works for you?
